spec/models/measurements_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'action_controller'
-require_relative '../app/controllers/application_controller'
-require_relative '../app/controllers/measurements_controller'
+require_relative '../../app/controllers/application_controller'
```

Comment:
Can you explain why this was added? What did this change?

Reply:
Had to move this file into a spec/models folder (previously in spec), so had to change this to correct mapping